File: w3/t4/Person.java
public class Person {
    public String firstName;
    public String lastName;
    private int age;
    public Person(String fname, String lname, int age) {
        this.firstName = fname;
        this.lastName = lname;
        this.age = age;
    }
    public int getAge() {
        return this.age;
    }
    public void ageUp() {
        this.age++;
    }
    public String getFullname() {
        return this.firstName + " " + this.lastName;
    }
    public void printFullname() {
        System.out.println(this.getFullname());
    }
}

File: w3/t6/person.dart
class Person {
    late String firstName;
    late String lastName;
    late int _age;
    Person(String fname, String lname, int age) {
        firstName = fname;
        lastName = lname;
        _age = age;
    }
    int getAge() {
        return _age;
    }
    void ageUp() {
        _age += 1;
    }
    String getFullname() {
        return firstName + " " + lastName;
    }
    void printFullname() {
        print(getFullname());
    }
}